### PR TITLE
qmmp: switch from ffmpeg_3 to ffmpeg

### PR DESCRIPTION
**Pull request by bjornfor**
###### Motivation for this change
Ref. issue #120705.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120771` at 27e8279a run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>qmmp</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, please check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/applications/audio/qmmp/default.nix:57:5:
```
   |
57 |     license = licenses.gpl2;
   |     ^
```
</li>
  </ul>
</details>

**Review by dotlambda — CHANGES_REQUESTED** [review]
License should be `gpl2Plus`.

**Comment by bjornfor** [comment]
> License should be `gpl2Plus`.

Done.

File: pkgs/applications/audio/qmmp/default.nix [hunk 1]
```diff
@@ -54,7 +54,7 @@ mkDerivation rec {
   meta = with lib; {
     description = "Qt-based audio player that looks like Winamp";
     homepage = "http://qmmp.ylsoftware.com/";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/qmmp/default.nix:56]:
```suggestion
    homepage = "https://qmmp.ylsoftware.com/";
```

File: pkgs/applications/audio/qmmp/default.nix [hunk 2]
```diff
@@ -54,7 +54,7 @@ mkDerivation rec {
...
-    license = licenses.gpl2;
+    license = licenses.gpl2Plus;
     platforms = platforms.linux;
     maintainers = [ maintainers.bjornfor ];
     repositories.svn = "http://qmmp.googlecode.com/svn/";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/qmmp/default.nix:60]: This URL returns 404.

Comment (bjornfor) [reply pkgs/applications/audio/qmmp/default.nix]: Fixed.

**Pull request merged**